### ISSUE TITLE
fix check-code-quality

### ISSUE THREAD

**Issue by kamilsk** [kamilsk/retry#98]
```
.PHONY: check-code-quality
check-code-quality: ARGS = \
	--exclude=".*_test\.go:.*error return value not checked.*\(errcheck\)$$" \
	--exclude="duplicate of.*_test.go.*\(dupl\)$$" \
	--vendor --deadline=2m ./...
check-code-quality: docker-tool-gometalinter
```

also define correct deadline: `time make check-code-quality`

blocked by:
- https://github.com/kamilsk/shared/issues/301
- https://github.com/kamilsk/shared/issues/298

**Comment by kamilsk** [comment]
extend git ignore

```
# go test and code quality report
*.out
```

add code quality report

```
.PHONY: code-quality-report
code-quality-report:
	time make check-code-quality | tail +7 | tee report.out | pbcopy
```
